CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # PayPal Android SDK Release Notes
 
+## unreleased
+* Breaking Changes
+  * CorePayments
+    * Access token integrations are now deprecated
```

Comment:
```suggestion
```

Don't think that line is necessary. Also I don't think "deprecated" is appropriate since it was removed entirely.